src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -256,11 +256,14 @@
         --network-plugin=azure will use an overlay network (non-VNET IPs) for pods in the cluster.
   - name: --network-policy
     type: string
-    short-summary: The Kubernetes network policy to use.
+    short-summary: Network Policy engine to use.
     long-summary: |
-        Using together with "azure" network plugin.
-        Specify "azure" for Azure network policy manager, "calico" for calico network policy controller, "cilium" for Azure CNI powered by Cilium.
-        Defaults to "" (network policy disabled).
+        Azure provides three Network Policy engines for enforcing network policies that can be used together with "azure" network plugin. The following values can be specified:
+          - "azure" for Azure Network Policy Manager,
+          - "cilium" for Azure CNI Overlay powered by Cilium,
```

Comment:
Product name is "Azure CNI Powered by Cilium" (works with both overlay and podsubnet)

Reply:
Fixed to "Azure CNI Powered by Cilium"

Reply:
still says Overlay?

Reply:
Sorry, fixed